### ISSUE TITLE
Cannot commit two classes with the same case insensitive name

### ISSUE THREAD

**Issue by jecisc** [pharo-vcs/iceberg#1014]
If you create a class named `AClass` and another named `Aclass`, you cannot commit them in a Tonel or FileTree repository because either:
- One will be committed and not the other
- One will be committed with the methods of both classes

**Comment by estebanlm** [comment]
I do not think this have an easy solution (or if possible), because of the case insensitive nature of some file systems.

**Comment by cdlm** [comment]
What about preventing creating, saving, or loading classes with case mismatches, basically treating them as a syntax error?

I think making the system case insensitive is too lax and only leads to more confusion. Better accept that there's nothing we can do about file systems, that case mismatches do not have legitimate use and therefore are an error, and prevent them early.
